### ISSUE TITLE
unexpected  loading 

### ISSUE THREAD

**Issue by songshitong** [peng8350/flutter_pulltorefresh#253]
version:
  pull_to_refresh: ^1.5.8
  flutter doctor
    
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.12.13+hotfix.8, on Mac OS X 10.15.2 19C57,
    locale zh-Hans)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
[✓] Android Studio (version 3.6)
[!] VS Code (version 1.42.1)
    ✗ Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter
[✓] Connected device (1 available)
```

description:
   When  soft keyboard  popup, there is an unexpected  loading on the top.

picture:
  normal:
  
![image](https://user-images.githubusercontent.com/26601878/76048062-9f94f680-5f9f-11ea-94b7-bbf17b3bdb58.png)

 err:
![image](https://user-images.githubusercontent.com/26601878/76048032-868c4580-5f9f-11ea-850e-a373a30a88a9.png)

**Comment by edTheGuy00** [comment]
Make sure the refresh controller has `initialRefresh` set to false if you don't want this behaviour `RefreshController(initialRefresh: false)`

when set to `true` the `SmartRefresher` widget will call `onRefresh` everytime the widget rebuilds, ie. when there are configuration changes such as when the keyboard pops up.

**Comment by peng8350** [comment]
fixed,this is the mistake when fix the change 1.13.6 forced change.
